config/settings.yml

```diff
@@ -29,7 +29,7 @@ redis:
     uri: <%= ENV.fetch("REDIS_SIDEKIQ_URI") { "redis://127.0.0.1:6379/1" } %>
   database:
     uri: <%= ENV.fetch("REDIS_DATABASE_URI") { "redis://127.0.0.1:6379/2" } %>
-matomo_id: 11
+matomo_id: 13
```

Comment:
@Smullz622 I would change this to be 38, and we'll leave the override in the config-repo!